website/docs/plugin/testing/acceptance-tests/ephemeral-resources.mdx

```diff
@@ -0,0 +1,274 @@
+---
+page_title: 'Plugin Development - Acceptance Testing: Ephemeral Resources'
+description: >-
+    Guidance on how to test ephemeral resources and data.
+---
+
+<Highlight>
+
+Ephemeral resource support is in technical preview and offered without compatibility promises until Terraform 1.10 is generally available.
+
+</Highlight>
+
+# Ephemeral Resources
+
+[Ephemeral Resources](/terraform/language/v1.10.x/resources/ephemeral) are an abstraction that allows Terraform to reference external data, similar to [data sources](/terraform/language/data-sources), without persisting that data to plan or state artifacts. The `terraform-plugin-testing` module exclusively uses Terraform plan and state artifacts for it's assertion-based test checks, like [plan checks](/terraform/plugin/testing/acceptance-tests/plan-checks) or [state checks](/terraform/plugin/testing/acceptance-tests/state-checks), which means that ephemeral resource data cannot be asserted using these methods alone.
+
+In addition to this, ephemeral resources have a different [lifecycle](/terraform/language/v1.10.x/resources/ephemeral#lifecycle) then resources or data sources and must be [referenced](/terraform/language/v1.10.x/resources/ephemeral#referencing-ephemeral-resources) as a dependency to a managed resource or data source to be invoked. This means for acceptance testing ephemeral resource logic, provider developers should use a real world example where the ephemeral resource would be referenced.
```

Comment:
TODO: This wording needs to be adjusted. I can't remember if this used to be the case, or if my brain fabricated it, but you definitely can just put an ephemeral resource in a config and run it, and it will execute `Open`

Reply:
Done ⚡ 4644758b79e5e1f91bd5ff65061f43e0076fce2e

I absolutely fabricated this 😆